### ISSUE TITLE
Documentation - cross links to the Wagtail user guide 

### ISSUE THREAD

**Issue by lb-** [wagtail/wagtail#11156]
### Pertinent section of the Wagtail docs

(Insert the permalink of the section heading in the Wagtail docs. Choose the most specific section / lowest-level heading to which your suggested change would apply. (In the docs, hover over the heading to see the permalink icon.))

### Details

(Write your answer here.)

**Comment by lb-** [comment]
Issue updated.

**Comment by NXPY123** [comment]
I think the django links use intersphinx mapping
```python
intersphinx_mapping = {
    "django": (
        "https://docs.djangoproject.com/en/stable/",
        "https://docs.djangoproject.com/en/stable/_objects/",
    )
}
```
We can't use that because the user guide is not built using sphinx right?

**Comment by lb-** [comment]
I think we could use the custom URL resolution feature though.

https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html#customising-external-url-resolution
https://myst-parser.readthedocs.io/en/latest/configuration.html

This way we could make a guide link with this syntax.

```md
[something something](guide:how-to-guides/manage-images)
```

OR

```
<guide:how-to-guides/manage-images>
```

---

As for the admonition part.

Combined with maybe the `tip` admonition we could do this.


```
```{tip} How to manage images
:class: guide
<guide:how-to-guides/manage-images>
```
```

We could then use styling on `.guide` to make this visually different. There may even be a configurable to add a different admonition type or use a different label for the `tip` admonition so it says something like "Guide...".

**Comment by NXPY123** [comment]
I think `myst_url_schemes` might work

**Comment by NXPY123** [comment]
I did try making `myst_url_schemes` a dictionary:
```python
myst_url_schemes = {
    "guide": "https://guide.wagtail.org/en-latest/{{path}}#{{fragment}}"
}
```

 But it keeps giving the following error:
 ```
 ERROR: myst configuration invalid: 'url_schemes' must be of type (<class 'list'>, <class 'tuple'>) (got {'guide': 'https://guide.wagtail.org/en-latest/{{path}}#{{fragment}}'} that is a <class 'dict'>).
 ```

**Comment by lb-** [comment]
Hmm ok. I'm not sure but I have seen that we may need to update our Myst Parser version. That could be why some of the configs are not working as expected. 

https://github.com/wagtail/wagtail/issues/11178 - raised this new issue to track the update.

**Comment by allcaps** [comment]
Wouldn't it be better to avoid dependency between the projects, but just copy over the image file to Wagtail docs? What is wrong with having images directly in the docs?

In Wagtail guide, images are just content, and might be less stable than you think. They can be deleted, or updated, changing the filename. Released versions of the Wagtail docs can't be updated. So we potentially create documentation with broken images.

I'd be more confident in

1. A stand-alone project that produces an immutable, stable image glossary. For both docs and guide to utilise.

2. Images in the Wagtail repo (after all, the user guide was part of the main docs)

3. Point from docs to the guide page URL. This has the benefit that it gives all context about the feature from the editor perspective.

4. Add a stable visual glossary to the guide reference section.

5. If the images must be in the guide, at least provide a stable resource locations. Maybe a dedicated screenshot modal/view with an URL that doesn't change. No delete, no filename change after updates. We might even come up with a way that we can update the media storage backend without altering the resource locations.

**Comment by allcaps** [comment]
@thibaudcolas tagging you as you have experience with creating the screenshots.

Are we updating the guide all the time? Or after a release?

Guess the docs need the resources beforehand.

**Comment by allcaps** [comment]
Just to clarify, I like the idea, but want to make sure we build a solution that works for us. And not increase the maintenance burden.

The guide people discussed creating and managing screenshots a couple of times. Using them in the docs is a new aspect to consider.

**Comment by allcaps** [comment]
@NXPY123 you'd need to supply a list:

myst_url_schemes = [{ "guide": "https://guide.wagtail.org/en-latest/{{path}}#{{fragment}}" }]

Note the [ ].

**Comment by NXPY123** [comment]
> @NXPY123 you'd need to supply a list:
> 
> myst_url_schemes = [{ "guide": "https://guide.wagtail.org/en-latest/{{path}}#{{fragment}}" }]
> 
> Note the [ ].

The error changed to this:
```
ERROR: myst configuration invalid: 'url_schemes[0]' must be of type <class 'str'> (got {'guide': 'https://guide.wagtail.org/en-latest/{{path}}#{{fragment}}'} that is a <class 'dict'>).
```

**Comment by NXPY123** [comment]
> Hmm ok. I'm not sure but I have seen that we may need to update our Myst Parser version. That could be why some of the configs are not working as expected.
> 
> #11178 - raised this new issue to track the update.

That was the issue: https://github.com/executablebooks/MyST-Parser/commit/879cfca4ee4d5f64892dd3cd5ecabe3050f4ba1a
The support was only added in 1.19.0

**Comment by NXPY123** [comment]
It works when you install the latest version of Myst Parser

**Comment by thibaudcolas** [comment]
I’d recommend holding off on doing this until after the Outreachy documentation project, or towards the end, so we take this into account as part of the project’s planning. There’s a good likelihood we’ll add a lot of documentation to the guide, which might mean having changing the structure of the content _a lot_. We’d be

**Comment by lb-** [comment]
Agreed. I'll add the label needs design decision - there also needs to be a clear direction on how we want this to work. Thanks for exploring this @NXPY123

**Comment by lb-** [comment]
MyST parser has been updated, so when it's time it should be quite easy to create these smarter links. 

https://github.com/wagtail/wagtail/pull/11194
